src/scenes/TradeHub/index.js

```diff
@@ -70,9 +59,11 @@ class TradeHub extends Component {
       this.setState({currentPrice});
       this.checkPrice(currentPrice);
     });
-  }
+  };
 
-  getCryptoList() {
+  removeSocket = endpoint => binance.websockets.terminate(endpoint);
+
+  getCryptoList = () => {
     fetch('https://api.binance.com/api/v1/exchangeInfo').then(res => res.json()).then((data) => {
```

Comment:
This is a hardcoded API path. If you're going to fetch it from multiple places, I can strongly recommend using a constant.

Reply:
I suggest we make some kind of configuration file, so it is easily editable.

Reply:
Yeah sure

Reply:
Like ApiConstants.js